client/components/EditUserForm/EditUserForm.tsx

```diff
@@ -0,0 +1,80 @@
+import './EditUserForm.scss'
+import React, { useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { User, Users, UserStatus } from '@meta/user'
+
+import { useAppDispatch } from '@client/store'
+import { useAssessment, useCycle } from '@client/store/assessment'
+import { UserManagementActions } from '@client/store/userManagement'
+import { useCountryIso } from '@client/hooks'
+import { useToaster } from '@client/hooks/useToaster'
+
+import Buttons from './Buttons'
+// import CountryRoles from './components/CountryRoles'
+import ProfilePicture from './ProfilePicture'
+import TextInputFields from './TextInputFields'
+
+const EditUserForm: React.FC<{ user: User }> = ({ user }) => {
+  const dispatch = useAppDispatch()
+  const { i18n } = useTranslation()
+  const { toaster } = useToaster()
+  const countryIso = useCountryIso()
+  const assessment = useAssessment()
+  const cycle = useCycle()
+
+  const [profilePicture, setProfilePicture] = useState<File>(null)
+
+  const saveUser = useCallback(() => {
+    if (!Users.validate(user).isError) {
+      dispatch(UserManagementActions.updateUser({ user, profilePicture })).then(() => {
+        dispatch(
+          UserManagementActions.getUsers({ countryIso, assessmentName: assessment.props.name, cycleName: cycle.name })
+        )
+        dispatch(UserManagementActions.setUserToEdit(null))
+        toaster.success(i18n.t('userManagement.userModified', { email: user.email }))
+      })
+    }
+  }, [assessment.props.name, countryIso, cycle.name, dispatch, i18n, profilePicture, toaster, user])
+
+  const changeUser = useCallback(
+    (value: string, key: string) => {
+      dispatch(
+        UserManagementActions.setUserToEdit({
+          ...user,
+          [key]: value,
+        })
+      )
+    },
+    [dispatch, user]
+  )
+
+  const deactivateUser = useCallback(() => {
+    dispatch(
```

Comment:
Does deactivate set in db active to false ?

Reply:
yes, it does